official/vision/modeling/backbones/resnet.py

```diff
@@ -293,12 +297,17 @@ def __init__(
         block_fn = nn_blocks.BottleneckBlock
       else:
         raise ValueError('Block fn `{}` is not supported.'.format(spec[0]))
+      use_first_projection = (
+          not (i == 0 and spec[0] == 'residual') and
+          not self._use_first_projection
```

Comment:
This statement is bit weird. Why the `use_first_projection` is opposite with  `not self._use_first_projection`?

Reply:
this is a mistake on my end : glad you caught it. Correcting.